src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/CartRule/Blocks/form.html.twig

```diff
@@ -0,0 +1,48 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{{ form_start(cartRuleForm) }}
+{{ form_errors(cartRuleForm) }}
+<div class="card">
+  <h3 class="card-header">
+    {{ 'Cart rule'|trans({}, 'Admin.Catalog.Feature') }}
+  </h3>
+  <div class="card-body">
+    {{ form_widget(cartRuleForm) }}
+    {% block cart_rule_form_rest %}
+      {{ form_rest(cartRuleForm) }}
+    {% endblock %}
```

Comment:
```suggestion
    {{ form_row(cartRuleForm) }}
```
Can't you do this instead? If the root form has an empty label this should work.

I'm starting to think maybe we should add a `CardType` form type, this would avoid duplicating this card thing all the time